src/plugins/data/public/search/expressions/__snapshots__/es_raw_response.test.ts.snap

```diff
@@ -0,0 +1,199 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`esRawResponse converts aggregations to table simple aggregation response 1`] = `
+Object {
+  "columns": Array [
+    Object {
+      "id": "2.buckets.key",
+      "meta": Object {
+        "field": "2.buckets.key",
+        "params": Object {},
+        "type": "string",
+      },
+      "name": "2.buckets.key",
+    },
+    Object {
+      "id": "2.buckets.doc_count",
+      "meta": Object {
+        "field": "2.buckets.doc_count",
+        "params": Object {},
+        "type": "number",
+      },
+      "name": "2.buckets.doc_count",
+    },
+    Object {
+      "id": "2.doc_count_error_upper_bound",
+      "meta": Object {
+        "field": "2.doc_count_error_upper_bound",
+        "params": Object {},
+        "type": "number",
+      },
+      "name": "2.doc_count_error_upper_bound",
+    },
+    Object {
+      "id": "2.sum_other_doc_count",
+      "meta": Object {
+        "field": "2.sum_other_doc_count",
+        "params": Object {},
+        "type": "number",
+      },
+      "name": "2.sum_other_doc_count",
+    },
+  ],
+  "meta": Object {
+    "source": "*",
+    "type": "esdsl",
+  },
+  "rows": Array [
+    Object {
+      "2.buckets.doc_count": 1033,
+      "2.buckets.key": "FEMALE",
+      "2.doc_count_error_upper_bound": 0,
+      "2.sum_other_doc_count": 0,
+    },
+    Object {
+      "2.buckets.doc_count": 944,
+      "2.buckets.key": "MALE",
+      "2.doc_count_error_upper_bound": 0,
+      "2.sum_other_doc_count": 0,
+    },
+  ],
+  "type": "datatable",
+}
+`;
+
+exports[`esRawResponse converts raw docs to table simple docs response 1`] = `
+Object {
+  "columns": Array [
+    Object {
+      "id": "order_date",
+      "meta": Object {
+        "field": "order_date",
+        "params": Object {},
+        "type": "object",
+      },
+      "name": "order_date",
+    },
+    Object {
+      "id": "products.created_on",
+      "meta": Object {
+        "field": "products.created_on",
+        "params": Object {},
+        "type": "object",
+      },
+      "name": "products.created_on",
+    },
+  ],
+  "meta": Object {
+    "source": "*",
+    "type": "esdsl",
+  },
+  "rows": Array [
+    Object {
+      "order_date": Array [
+        "2020-07-13T09:27:22.000Z",
+      ],
+      "products.created_on": Array [
+        "2016-12-12T09:27:22.000Z",
+        "2016-12-12T09:27:22.000Z",
+      ],
```

Comment:
This row value is not useful for visualizations, since it's an array. In the spacetime work that Joe and I did, this was the most important case to figure out. Both of us chose to unroll these nested arrays into multiple rows.

Reply:
this response handling is copied from joe's space time. I am hoping we can get elasticsearch to return meta information with responses and thus address this, but if you have a simple solution that would make this parsing better i am open for suggestions.

Reply:
Something's definitely not right here as these rows only contain the `order_date` and `products.created_on` fields, nothing else. It should at least map all of the unambiguous fields as well, not sure what's happening here.

Reply:
Agreed that we should follow up with this later. It might need special handling